core/src/main/scala/kafka/server/BrokerServer.scala

```diff
@@ -22,7 +22,6 @@ import java.util.concurrent.atomic.AtomicBoolean
 import java.util.concurrent.locks.ReentrantLock
 import java.util.concurrent.{CompletableFuture, TimeUnit, TimeoutException}
 import java.net.InetAddress
-
```

Comment:
Can we leave this blank line? It seems like the standard is to separate out Java includes. Let's not ping/pong on this

Reply:
Sure, I didn't meant to remove this (at least, I don't think I did)